### ISSUE TITLE
diff with record_store 5.0.x returns unexpected diffs for DNSimple

### ISSUE THREAD

**Issue by charlescng** [Shopify/record_store#52]
Related to: https://github.com/Shopify/record_store/pull/47

For example:
```
bin/record-store diff
...
Zone: jadedpixel.com
--------------------
Provider: DNSimple
Add:
 - [TXTRecord] jadedpixel.com. 600 IN TXT "spycloud-domain-verification=39c758f7-86e8-4d72-a7c0-39fb9d3032e4"
====================
...
```
The record exists from the DNSimple UI:
![image](https://user-images.githubusercontent.com/6034184/36225087-928034a8-1197-11e8-8091-efe87027e9d0.png)

EDIT: Due to fog/fog-dnsimple#4, diff is broken for DNSimple with 5.0.x releases.

Running the diff with record_store 4.0.7 returns no diffs.

/cc @weppos @es

**Comment by weppos** [comment]
@ChanChar how many records do the zone have? Can it be caused by https://github.com/fog/fog-dnsimple/issues/4?

**Comment by charlescng** [comment]
That particular zone has 32 records.

**Comment by weppos** [comment]
That is probably the cause then. By default our pagination returns 30 records.
I have that issue on my bucket list. Sadly, as the fog gem is not based on our Ruby API client, it requires some extra work to make it compatible.

**Comment by charlescng** [comment]
Maybe it's a good time to ditch fog and use your gem directly.

**Comment by albiere** [comment]
I'll spend some time to replace the fog gem with the dnsimple one.

**Comment by sbfaulkner** [comment]
I'm pretty certain this was fixed with #54 - right @albiere ?

**Comment by albiere** [comment]
@sbfaulkner we're running with the dnsimple gem and we're all good now.
